src/components/input.tsx

```diff
@@ -6,11 +6,22 @@ interface Props {
   placeholder?: string
   svg?: HTMLElement
   value?: string
+  text_align?: string
```

Comment:
変数名はスネークケースではなくキャメルケース

```suggestion
  textAlign?: string
```